client/app/visualizations/table/renderer.less

```diff
@@ -103,6 +103,10 @@
 
   /* START table x scroll */
   .dashboard-widget-wrapper:not(.widget-auto-height-enabled) & {
+    position: absolute;
+    left: 0;
+    top: 0;
+    width: 100%;
```

Comment:
I'm not a big fan of this unless absolutely (pun intended) necessary.
Instead you can just add `overflow-x: auto` to:
https://github.com/getredash/redash/blob/b2c13d3d6bb473ba9969c82a132f2dc09b4304a3/client/app/pages/dashboards/dashboard.less#L69-L72

Reply:
@ranbena Tried your suggestion and seems it works. Can you please briefly test it to confirm that your table scroll feature works as expected (with and without filters)? I tested it of course, but want to be sure that I didn't miss anything. Thanks!

Reply:
Sure. I'll be able to within the hour.

Reply:
Okay, this variant does not work on test environment (not sure why). I'll revert it

Reply:
![image](https://user-images.githubusercontent.com/12139186/64946675-0f8db400-d87c-11e9-9408-2ca51c939b7b.png)